### PR TITLE
ROX-25620: Update Konflux task repos

### PR DESCRIPTION
**Pull request by msugakov**
### Description

Per title.
Goes after https://github.com/stackrox/stackrox/pull/12337

### User-facing documentation

- [x] CHANGELOG is updated **OR** update is not needed
- [x] [documentation PR](https://spaces.redhat.com/display/StackRox/Submitting+a+User+Documentation+Pull+Request) is created and is linked above **OR** is not needed

### Testing and quality

- [x] the change is production ready: the change is GA or otherwise the functionality is gated by a feature flag
- [ ] CI results are inspected

#### Automated testing

No change to automated tests.

#### How I validated my change

Only CI (excluding EC).

### CODE REVIEW

File: .tekton/determine-image-tag-task.yaml [hunk 1]
```diff
@@ -31,7 +31,7 @@ spec:
       name: workdir
   steps:
   - name: use-trusted-artifact
-    image: quay.io/redhat-appstudio/build-trusted-artifacts:latest@sha256:4e39fb97f4444c2946944482df47b39c5bbc195c54c6560b0647635f553ab23d
+    image: quay.io/redhat-appstudio/build-trusted-artifacts:latest@sha256:601e0999ae062e5ee666538d651af7893a75e415b7952c85caa8a4452501029a
```
Comment (tommartensen) [inline .tekton/determine-image-tag-task.yaml:34]: Is this image not available from `quay.io/konflux-ci`?

Comment (msugakov) [reply .tekton/determine-image-tag-task.yaml:34]: Nope, I tried both
`quay.io/konflux-ci/build-trusted-artifacts:latest` and
`quay.io/konflux-ci/tekton-catalog/build-trusted-artifacts:latest`

**Review by tommartensen — APPROVED** [review]

**Pull request merged**